src/main/kotlin/de/cyface/collector/model/Measurement.kt

```diff
@@ -128,8 +128,6 @@ data class Measurement(
         val geoJson = toGeoJson(deviceMetaData, applicationMetaData, measurementMetaData, attachmentMetaData)
 
         geoJson
-            .getJsonArray("features")
-            .getJsonObject(0)
```

Comment:
Instead of storing the measurement metadata as `FeatureCollection` with always exactly one feature I thought it is cleaner to just store the `Feature` directly, which is legit as long as we do not expect the geometry to consist of multiple features any time in the future.

Even when we store the whole track, we can use the one MultiLine-feature as now. Only if we would decide at some point that we store each tracks' start and end point in the metadata as geometry, we would need multiple features.